### DISCUSSION TITLE
Your instructions to setup a project is anti-noob, very frustrating...

### DISCUSSION THREAD

**Discussion by Mellifico** [jycouet/kitql#340] [Docs Discussion] [answered]
"Get starded" for kitql lacks a ton of informations, if I compare to your demo project. I follow exactly **all** your instructions and it failed like :  Error when evaluating SSR module /src/lib/graphql/$kitql/_appModules.ts: failed to import "../../modules/__init"

Looking at the demo project, it contains a ton of folders and files you did not explain in your "get started". Are we suppose to guess what we need to write and what will be created automatically ?

Maybe you could help me a bit.
Thanks

PS: I'm french

**Comment by jycouet** [comment, marked as answer]
Yes, you are right; it's missing the "modules" explanation.
I should write down something about it 👍

Maybe we can chat on Discord? I'm `jycouet#0711`.

**Reply by Mellifico** [reply]
Thank you !
Maybe we can chat in french, if possible ?
I will contact you on discord sooner or later.
See U,
Phil

**Reply by mrcleanandfresh** [reply]
One of the things I felt it was missing was how to use `@envelop/*` plugins. I am trying to use some of them, but I keep getting type errors when trying to put them in the plugin array. I also have this mystery generated file that is called `ctx.ts` with the contents of:

```typescript
// No DbProvider found
export {}
```

I'm using Prisma for my DbProvider and I added it to the `getContext` so I have no clue (and I'm _not_ a new developer) what I'm supposed to do to define the context, or use these plugins. I would love to help contribute to the docs, but I am working off of very little information and a black box of voodoo! hah. It's a very great tool and I love it a lot, but I just don't understand how it works, which makes it incredibly frustrating to debug. Especially since SvelteKit does not support server side debugging yet.

I was able to get through the docs and get up and running but I basically just copy/pasted files from the StackBlitz example, not knowing what any of it was for, and I didn't realize that the tool could run even without the files being resolved in the `$kitql` folder (not generated yet). I should have assumed, since it's gotta work on CI. Anyway, I have to concur with the original post.

Where is this discord server? I'd like to get some help with some of my questions. Thanks for a great tool!

**Reply by jycouet** [reply]
I have to say that I'm sorry for the bad experience because I didn't spend time documenting all this.
I have a few "big" projects working with envelop modules & Prisma through `kitql`. Pretty much what you are looking for.

You can join Houdini discord here: https://discord.gg/K33YbQqB
And/or contact me directly on jycouet#0711.

It would be awesome to show you how I'm using it and spend the time to explain it in the doc.
Speak to you soon

**Comment by ZeroPie** [comment]
Just wanted to say: I also decided to skip it because of the (lack of) documentation.
